e2e-polybft/cardanofw/test-cardano-chain.go

```diff
@@ -450,3 +490,11 @@ func (ec *TestCardanoChain) submitTx(
 		return txHash, nil
 	}, infracommon.WithRetryCount(retryCount), infracommon.WithRetryWaitTime(retryWaitTime))
 }
+
+func GetExchangeRate(sourceChainID string, destinationChainID string) (float64, error) {
```

Comment:
remove this... we will later add dynamic exchange rate through some interface implementation